workers/unity/Assets/Playground/Scripts/MonoBehaviours/DeleteCubeCommandReceiver.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using Generated.Playground;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+#pragma warning disable 649
+
+public class DeleteCubeCommandReceiver : MonoBehaviour
+{
+    [Require] private WorldCommands.WorldCommandRequestSender worldCommandRequestSender;
+    [Require] private WorldCommands.WorldCommandResponseHandler worldCommandResponseHandler;
+
+    [Require] private CubeSpawner.Requirables.CommandRequestHandler commandRequestHandler;
+    [Require] private CubeSpawner.Requirables.Writer writer;
+
+    private WorldCommandHelper worldCommandHelper;
+
+    public void OnEnable()
+    {
+        commandRequestHandler.OnDeleteSpawnedCubeRequest += OnOnDeleteSpawnedCubeRequest;
+
+        worldCommandHelper = new WorldCommandHelper(
+            worldCommandRequestSender,
+            worldCommandResponseHandler);
+    }
+
+    public void OnDisable()
+    {
+        worldCommandHelper.Dispose();
```

Comment:
check for null: `worldCommandHelper?.Dispose()`

Reply:
This should never happen, but I'll add the null check just in case because it will probably happen